route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -47,7 +47,7 @@ namespace route_following_plugin
 
         // read ros parameters
         pnh_->param<double>("minimal_plan_duration", min_plan_duration_, 16.0);
-        pnh_->param<std::string>("lane_change_plugin", lane_change_plugin_);
+        pnh_->param<std::string>("lane_change_plugin", lane_change_plugin_, "CooperativeLaneChangePlugin");
```

Comment:
If this wasn't being loaded properly before and had no default value how was it ever working?

Reply:
I dont think it was. The variable is initialized in the header, so it was using that value as a constant